public_html/dataView/allfiles.php

```diff
@@ -53,13 +55,11 @@
 		$files = array();
 
 		foreach ($object['data'][$field_name]['files']['archive'] as $file) {
-
-			$files[] = sprintf("%s",$file['name']);
-
+			$files[] = $file['name'];
 		}
 
 		$files           = implode(" ",$files);
-		$destinationFile = sys_get_temp_dir()."/".time().".".$type;
+		$destinationFile = sprintf("%s/%s.%s",mfcs::config('mfcsDownloadAllDir'),time(),$type);
 
 		if ($type == "zip") {
```

Comment:
I'm curious, would it be better to do something similar to the export scripts where you use symbolic linking, follow the links, and then zip the file following the symbolic links.